.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Description / Purpose
+
+Please include a description of the purpose and approach of this pull request's commits.
+This area should provide an overview of why the work is happening and, within reason, should include relevant links to the ticket, product/design briefs, and engineering plans. We shouldn’t assume familiarity with the history of this work. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Types of changes
+
+What types of changes does your code ?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
```

Comment:
Link should be updated to CONTRIBUTING.md for this repo

Reply:
Is there are CONTRIBUTING.md in this repo? 😄

Reply:
Don't think there's a CONTRIBUTING.md file yet, will update the link when it exists.